.mergify.yml

```
@@ -0,0 +1,8 @@
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - "#approved-reviews-by>=1"
+      - label!="WIP PR"
```

add a condition that the title is not containing [WIP]. this way also contributors. that cannot add labels are able to block the merge 

I think it's something like this
```suggestion
    conditions:
      - "#approved-reviews-by>=1"
      - label!="WIP PR"
      - -title~=\[WIP\]
```